canonical-web-and-design/ubuntu.com
# Tutorials sort filter

## Done

[List of work items including drive-bys]

## QA

- Go to `/tutorials`
- See that there is a topics filter that filters the tutorials by the selected topic
- See that there is a difficulty filter that sorts the tutorials by difficulty

Fixes #6211, #6363
